internal/archive/credentials.go

```diff
@@ -0,0 +1,304 @@
+package archive
+
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"io"
+	"net/url"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+)
+
+// This file defines findCredentials() function for searching repository
+// credentials in Apt configuration, see
+// https://manpages.debian.org/testing/apt/apt_auth.conf.5.en.html.
```

Comment:
Can we please drop this comment and move any relevant information to the function itself?  (this files defines all the functions/types/etc that are defined in this file).

Reply:
Removed. Other function are already documented with the information that was here.